GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs

```diff
@@ -240,6 +240,7 @@ private static void GitHubItem_Click(object sender, EventArgs e)
 
         private static void IssuesItem_Click(object sender, EventArgs e)
         {
+            UserEnvironmentInformation.CopyInformation();
```

Comment:
I would prefer to not copy here, the user may have copied something already
(in Browse too)

Reply:
Note the issue template change.  The user can recopy the thing they needed.  When they fill in the issue, they will see the note about what is in the clipboard.  I was looking for a toast notification that would let them know environment info was copied.  A non modal messagebox as it were.

Reply:
I don't think we need another message box here. Keep it simple.